homeassistant/components/vera.py

```diff
@@ -43,9 +44,14 @@
         vol.Required(CONF_CONTROLLER): cv.url,
         vol.Optional(CONF_EXCLUDE, default=[]): VERA_ID_LIST_SCHEMA,
         vol.Optional(CONF_LIGHTS, default=[]): VERA_ID_LIST_SCHEMA,
+        vol.Optional(CONF_CLIMATE, default="climate"): cv.string
     }),
 }, extra=vol.ALLOW_EXTRA)
 
+VERA_COMPONENTS = [
+    'binary_sensor', 'sensor', 'light', 'switch', 'lock'
```

Comment:
Not the way I wrote it... But I'll be changing it to never add as thermostat so I will be adding climate here as part of those changes. Currently Line 92 added either climate or thermostat based on config, but I'll be removing that an any reference of thermostat based on these comments.